spec/system/articles/user_visits_an_article_spec.rb

```diff
@@ -15,8 +15,9 @@
     expect(page).to have_content(article.title)
   end
 
-  it "shows comments", js: true do
+  it "shows non-negative comments", js: true do
     create_list(:comment, 3, commentable: article)
+    create :thumbsdown_reaction, reactable: create(:comment, commentable: article)
 
     visit article.path
```

Comment:
**Suggestion:** I took a quick look at the failing spec and this small tweak seemed to solved the issue in creating the thumbs down reaction

```suggestion
    admin = create(:user, :admin)
    create(:thumbsdown_reaction, reactable: create(:comment, commentable: article), user: admin)
    sign_out user
    visit article.path
```